src/content/en/updates/2019/08/barcode-detection.md

```diff
@@ -0,0 +1,128 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The Barcode Detection API allows for barcodes to be detected in images.
+
+{# wf_updated_on: 2019-08-05 #}
+{# wf_published_on: 2019-08-01 #}
+{# wf_tags: capabilities,shape-detection,progressive-web-apps,webapp #}
+{# wf_featured_image: /web/updates/images/generic/timeline.png #}
+{# wf_featured_snippet: The Barcode Detection API allows for barcodes to be detected in images. #}
+{# wf_blink_components: Blink>ImageCapture #}
+
+# Shipping the Barcode Detection Part of the Shape Detection API {: .page-title}
+
+{% include "web/_shared/contributors/thomassteiner.html" %}
+
+<div class="clearfix"></div>
+
+Success: We were working the specification for this API as part of the
```

Comment:
Shouldn't it be "We were working **on** the specification for this API"?

Reply:
To my ears as well, but I took this from @petele, who is a native speaker.

Reply:
I think the "on" is needed indeed.

Reply:
If you google "work on the specification" (with double quotes), you get about 7,510,000 results, two of which are from w3.org, with exact text match. While if you google "work the specification" (with double quotes), you get about 1,740,000 results, many of which include this text, but with a period or comma after "work", in other words, you find this match but as part of separate sentences.

Reply:
You generally work **on** something, not work somethin :)

Reply:
If you google "work on the specification" (with double quotes), you get about 7,510,000 results, two of which are from w3.org, with exact text match. While if you google "work the specification" (with double quotes), you get about 1,740,000 results, many of which include this text, but with a period or comma after "work", in other words, you find this match but as part of separate sentences.